google/resource_compute_instance.go

```diff
@@ -1003,16 +1006,15 @@ func resourceComputeInstanceRead(d *schema.ResourceData, meta interface{}) error
 		return fmt.Errorf("Expected %d disks, API returned %d", expectedDisks, len(instance.Disks))
```

Comment:
Not right now- because we're relying so much on what's written in state, we can't really support that case without a fair amount of effort. It'll probably be easier once `disks` is gone for good, so it'll probably be worth a shot then.